### PR TITLE
Adding details to PACKER_BUILD_NAME to have it clear in the documentation how the variable is created

### PR DESCRIPTION
**Pull request by ruzickap**
Adding details to `PACKER_BUILD_NAME` to have it clear in the documentation how the variable is created.

It's not clear from the section about `PACKER_BUILD_NAME` ...

Prevents from creating bugs like: #6016

### CODE REVIEW

File: website/source/docs/post-processors/shell-local.html.md [hunk 1]
```diff
@@ -84,7 +84,9 @@ commonly useful environmental variables:
 
 -   `PACKER_BUILD_NAME` is set to the name of the build that Packer is running.
     This is most useful when Packer is making multiple builds and you want to
-    distinguish them slightly from a common provisioning script.
+    distinguish them slightly from a common provisioning script. You can find
```
Comment (mwhooker) [inline website/source/docs/post-processors/shell-local.html.md]: Maybe instead we can just add a link?

> `PACKER_BUILD_NAME` is set to the [name of the build](/docs/templates/builders.html) that Packer is running.

**Comment by ruzickap** [comment]
I agree with your point. I changed it and remove the qemu example (I'll create new PR for it - and we can discuss it there...).

**Comment by ruzickap** [comment]
I made a mess in this PR - let me create new one...

**Comment by ruzickap** [comment]
Please follow #6023

**Pull request closed without merging**